.circleci/config.yml

```diff
@@ -326,6 +326,13 @@ commands:
           # Stop running Puppeteer end-to-end test if code in relevant directories is unchanged.
           dir_names: "ui e2e .circleci"
 
+  block-concurrent-build:
+    description: "Block this job until all previous builds has finished"
+    steps:
+      - run:
+          command: bash .circleci/wait-until-previous-builds-complete.sh
+          name: "Run wait-until-previous-builds-complete.sh"
```

Comment:
Would prefer a normal sentence here, otherwise this is not providing much value over just reading the bash command itself